### PR TITLE
hew heading

### PR DESCRIPTION
**Pull request by dowen12**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: .specific/pre-reqs.adoc [hunk 1]
```diff
@@ -7,3 +7,7 @@ _Describe any setup required in the AWS account prior to template launch_
 ==== Prepare your {partner-company-name} account
 
 _Describe any setup required in the partner portal/account prior to template launch_
+
+==== Prepare for the deployment
+
+_Describe any preparation required to complete the product build, such as obtaining licenses or placing files in S3_
```
Comment (andrew-glenn) [inline .specific/pre-reqs.adoc:13]:
```suggestion
_Describe all preparation required to complete the product build, such as obtaining licenses or placing files in S3_
```

File: .specific/pre-reqs.adoc [hunk 2]
```diff
@@ -7,3 +7,7 @@ _Describe any setup required in the AWS account prior to template launch_
 ==== Prepare your {partner-company-name} account
 
 _Describe any setup required in the partner portal/account prior to template launch_
+
+==== Prepare for the deployment
```
Comment (andrew-glenn) [inline .specific/pre-reqs.adoc:11]: This is a test of vscode stuff

File: .specific/pre-reqs.adoc (hunk 1 above)
Comment (dowen12) [reply .specific/pre-reqs.adoc:13]: looks good

File: .specific/_settings.adoc [hunk 3]
```diff
@@ -3,7 +3,7 @@
 :partner-company-name: Example Company Name, Ltd.
 :doc-month: May
 :doc-year: 2020
-:partner-contributors: John Doe and Jane Doe
+:partner-contributors: John Doe and Jane Doe - {partner-company-name}
```
Comment (andrew-glenn) [inline .specific/_settings.adoc:6]: :partner-stuff:

Comment (andrew-glenn) [reply .specific/_settings.adoc:6]: sdfs

File: introduction.adoc [hunk 4]
```diff
@@ -9,7 +9,7 @@ ifndef::production_build[]
 endif::production_build[]
 [.text-center]
 *{doc-month}, {doc-year}* +
-_{partner-contributors} - {partner-company-name}_ +
+_{partner-contributors}_ +
```
Comment (dowen12) [inline introduction.adoc:12]: @andrew-glenn would need to change here too

**Comment by andrew-glenn** [comment]
```diff
diff --git a/.specific/_settings.adoc b/.specific/_settings.adoc
index f4b63c8..be13f82 100644
--- a/.specific/_settings.adoc
+++ b/.specific/_settings.adoc
@@ -5,7 +5,7 @@
 :doc-year: 2020
 :partner-contributors: John Doe and Jane Doe - {partner-company-name}
 :quickstart-contributors: Jim Smith - AWS Global Partner SA, AWS + Joe Jones - Technical Product Manager, AWS
-:deployment_time: 30 minutes / 1 hour
+:deployment_time: 30 minutes / 1 hour / 1 coffee break
 :default_deployment_region: us-east-1
 // Uncomment these two attributes if you are leveraging
 // - an AWS Marketplace listing.

```

**Comment by andrew-glenn** [comment]
```diff
diff --git a/.specific/pre-reqs.adoc b/.specific/pre-reqs.adoc
index f08b5d9..40e176d 100644
--- a/.specific/pre-reqs.adoc
+++ b/.specific/pre-reqs.adoc
@@ -8,6 +8,7 @@ _Describe any setup required in the AWS account prior to template launch_
 
 _Describe any setup required in the partner portal/account prior to template launch_
 
+AWS RDS stuff
 ==== Prepare for the deployment
 
 _Describe any preparation required to complete the product build, such as obtaining licenses or placing files in S3_
\ No newline at end of file

```

**Comment by andrew-glenn** [comment]
```diff
diff --git a/.specific/_settings.adoc b/.specific/_settings.adoc
index f4b63c8..430c9b1 100644
--- a/.specific/_settings.adoc
+++ b/.specific/_settings.adoc
@@ -6,7 +6,7 @@
 :partner-contributors: John Doe and Jane Doe - {partner-company-name}
 :quickstart-contributors: Jim Smith - AWS Global Partner SA, AWS + Joe Jones - Technical Product Manager, AWS
 :deployment_time: 30 minutes / 1 hour
-:default_deployment_region: us-east-1
+:default_d,mneployment_region: us-east-1
 // Uncomment these two attributes if you are leveraging
 // - an AWS Marketplace listing.
 // Additional content will be auto-generated based on these attributes.

```

**Comment by andrew-glenn** [comment]
```diff
diff --git a/.specific/pre-reqs.adoc b/.specific/pre-reqs.adoc
index f08b5d9..8701d9d 100644
--- a/.specific/pre-reqs.adoc
+++ b/.specific/pre-reqs.adoc
@@ -8,6 +8,6 @@ _Describe any setup required in the AWS account prior to template launch_
 
 _Describe any setup required in the partner portal/account prior to template launch_
 
-==== Prepare for the deployment
+==== Prepare for the deployment to work
 
 _Describe any preparation required to complete the product build, such as obtaining licenses or placing files in S3_
\ No newline at end of file

```

**Pull request merged**